draft-ietf-aipref-attach.md

```diff
@@ -149,24 +149,20 @@ Embedding preferences is expected to be an effective means
 of associating preferences with content,
 because it ensures that metadata is always associated with content.
 
-The main challenge with embedding is that
-a different method is needed for each content type.
-That is,
-a different means of conveying preferences
-needs to be defined for each audio, documents, images, video,
-or other content format.
+Each content type - whether audio, documents, images, video, or other - 
+embeds metadata in different ways, however, in most cases it is possible to 
+embed the same metadata, just in different places. This is, for example, how XMP (ISO 16684) 
```

Comment:
Separately, it occurs that maybe we want to acknowledge the potential for the encoding to differ between formats.